airflow/api_connexion/parameters.py

```diff
@@ -89,6 +89,13 @@ def wrapped_function(*args, **kwargs):
     return format_parameters_decorator
 
 
+def apply_array_filter(query, key, values):
```

Comment:
Could you also add type annotation to this function?

Reply:
Added